sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -611,6 +611,9 @@ class LinkedEntity(DictMixin):
     :ivar data_source: Data source used to extract entity linking,
         such as Wiki/Bing etc.
     :vartype data_source: str
+    :param str bing_id: Bing unique identifier of the recognized entity. Use in conjunction
```

Comment:
Also should we add that sphinx directive for properties that are only available for certain versions?

```suggestion
    :ivar str bing_id: Bing unique identifier of the recognized entity. Use in conjunction
```

Reply:
good point, i'll add that, it might not look great but at least we'll have it. thanks!

Reply:
We have other properties that should have this versionadded docstring too, have created an issue [here](https://github.com/Azure/azure-sdk-for-python/issues/13447) trackcing that